client/state/comments/selectors/comments-fetching-status.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { size } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { fetchStatusInitialState } from 'state/comments/reducer';
+import { getPostCommentItems } from 'state/comments/selectors/get-post-comment-items';
+import { getStateKey } from 'state/comments/utils';
+
+import 'state/comments/init';
+
+export function commentsFetchingStatus( state, siteId, postId, commentTotal = 0 ) {
+	const fetchStatus =
+		state.comments.fetchStatus?.[ getStateKey( siteId, postId ) ] ?? fetchStatusInitialState;
```

Comment:
Won't `fetchStatus` always be there if the `comments` tree exists?

Reply:
Oops, missed this one! Thanks!